mappings/net/minecraft/network/SplitterHandler.mapping

```diff
@@ -1,5 +1,12 @@
 CLASS net/minecraft/unmapped/C_hoasxhkp net/minecraft/network/SplitterHandler
-	METHOD decode decode (Lio/netty/channel/ChannelHandlerContext;Lio/netty/buffer/ByteBuf;Ljava/util/List;)V
+	FIELD f_bmaoystr MAX_LENGTH_BYTES I
+	FIELD f_rvhsntis helperBuf Lio/netty/buffer/ByteBuf;
+	METHOD decode (Lio/netty/channel/ChannelHandlerContext;Lio/netty/buffer/ByteBuf;Ljava/util/List;)V
 		ARG 1 context
 		ARG 2 buffer
 		ARG 3 output
+	METHOD handlerRemoved0 (Lio/netty/channel/ChannelHandlerContext;)V
```

Comment:
```suggestion
	METHOD handlerRemoved (Lio/netty/channel/ChannelHandlerContext;)V
```
?

Reply:
automapped!

Reply:
Why was this reverted?

Reply:
this is not a name for the method, this is its obf name

Reply:
oooh, oops